src/explore-education-statistics-frontend/src/modules/find-statistics/components/FilterModals.tsx

```diff
@@ -0,0 +1,100 @@
+import Button from '@common/components/Button';
+import Modal from '@common/components/Modal';
+import SummaryList from '@common/components/SummaryList';
+import SummaryListItem from '@common/components/SummaryListItem';
+import { ThemeSummary } from '@common/services/themeService';
+import styles from '@frontend/modules/find-statistics/components/FilterModals.module.scss';
+import React from 'react';
+
+export const ReleaseTypesModal = ({
+  open,
+  onClose,
+}: {
+  open: boolean;
+  onClose: () => void;
+}) => (
+  <Modal
+    className="govuk-!-width-one-half"
+    open={open}
+    title="Release types guidance"
+    onExit={onClose}
+  >
+    <p>
+      This is a description list of the different types of publication we
+      currently publish.
+    </p>
+    {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+    <div className={styles.content} tabIndex={0}>
```

Comment:
Does this need to be tabbable?

Is it related to the overflow styling? Why not allow the modal fill the entire screen and just tweak it so that the modal can be scrolled?